src/tests/functional/shared_test_classes/src/base/utils/generate_static_shapes.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Please move the file as `generate_static_shapes.hpp` to `src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/utils`